src/test/frontend/db/model_test.cljs

```diff
@@ -0,0 +1,36 @@
+(ns frontend.db.model-test
+  (:require [frontend.db.model :as model]
+            [frontend.db.conn :as conn]
+            [datascript.core :as d]
+            [frontend.db-schema :as schema]
+            [frontend.handler.repo :as repo-handler]
+            [cljs.test :refer [deftest is are testing]]))
+
+(defonce test-db "test-db")
+
+(defn- run-db!
+  []
+  (conn/start! nil test-db))
+
+(deftest test-page-alias-set
+  []
+  (run-db!)
+  (let [files [{:file/path "a.md"
+                :file/content "---\ntitle: a\nalias: [[b]]\n---"}
```

Comment:
Might be worth adding other test cases like aliases without `[[]]`, and more than one alias per page.

Reply:
Good suggestion, will do!
I'm trying to fix the CI issue now.

Reply:
Added more tests.